### DISCUSSION TITLE
Problem with installation inside the container

### DISCUSSION THREAD

**Discussion by jnlopez7** [Keyfactor/ejbca-ce#334] [General]
I can't install anything inside the container
The OS inside the container is a Almalinux, but dnf or yum doesn't work?
I need to install a text editor and I can't...

Besides that, what is the volume that should be persistent from the EJBCA ?

**Comment by primetomas** [comment]
If you are trying to change things inside the container, you are not using containers as intended. Containers should be ephemeral, can be started and stopped at any time and when stopped they go away, and you can pull a new container easily for upgrades.

What are you trying to achieve? Perhaps you can find some hints here:
https://github.com/Keyfactor/keyfactorcommunity
